terraform/variables/prod-intl.tfvars

```diff
@@ -119,7 +119,15 @@ default_aggregation_grace_period = "4h"
 default_peer_share_processor_manifest_base_url = "storage.googleapis.com/prio-enpa-g-prod-manifests"
 default_portal_server_manifest_base_url        = "manifest.global.enpa-pha.io"
 
-enable_key_rotation_localities = ["*"]
+# Put all localities in dry-run mode
+enable_key_rotation_localities = [""]
+
+packet_encryption_key_rotation_policy = {
+  create_min_age   = "48h"   // 48 hours = 2 days. The affected keys were generated Friday Dec 2.
+  primary_min_age  = "5m"    // 0 seconds, promote to primary immediately
```

Comment:
This comment is out of date

Reply:
So it is! I'm going to leave it as-is since rebasing and re-running CI across the three PRs isn't worth the hassle, since this code shouldn't (🤞🏻) be on `main` past this afternoon anyway.